From 2fa09bb0e250493ca21d4eaabc551f0fc193b4d1 Mon Sep 17 00:00:00 2001
From: Zhou weixin <zwx@rock-chips.com>
Date: Sun, 25 Feb 2018 10:43:14 +0800
Subject: [PATCH] arm64: dts: rockchip: set vcc_3v0 on and vcc1v8_dvp off in
 sleep mode for rk3326-863-lp3-v10

vcc_3v0: supply for wifi bt module, should keep enable in sleep mode.
vcc1v8_dvp: supply for camera, should be off in sleep mode, otherwise,
increase current consumption.

Change-Id: Idfd10f7b8f6fca6db1760e429acc9b215a8cf595
Signed-off-by: Weixin Zhou <zwx@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
index aea2bf7a0c3f..a7ac9b8e14fb 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dts
@@ -319,7 +319,7 @@
 				regulator-initial-mode = <0x1>;
 				regulator-name = "vcc_3v0";
 				regulator-state-mem {
-					regulator-off-in-suspend;
+					regulator-on-in-suspend;
 					regulator-suspend-microvolt = <3000000>;
 				};
 			};
@@ -424,7 +424,7 @@
 
 				regulator-name = "vcc1v8_dvp";
 				regulator-state-mem {
-					regulator-on-in-suspend;
+					regulator-off-in-suspend;
 					regulator-suspend-microvolt = <1800000>;
 				};
 			};
-- 
2.35.3

